spec/spec_helper.rb

```diff
@@ -43,20 +42,12 @@
 
   config.include FailHelpers
   config.include ClassHelpers
-
-  Aws.config.update(stub_responses: true) if defined?(::Aws)
 end
 
 if defined?(::ActiveRecord)
   require 'support/active_record'
-elsif defined?(::Mongoid)
-  require 'support/mongoid'
-elsif defined?(::Sequel)
-  require 'support/sequel'
 else
   RSpec.configure do |config|
-    %i[orm mongoid active_record sequel].each do |group|
-      config.filter_run_excluding(group)
-    end
+    config.filter_run_excluding(:orm)
```

Comment:
I meant we are not interested in running specs w/o ActiveRecord, arent' we?
Let's remove this `if defined?` block completely, leaving only unconditional `require 'support/active_record'`.
We also can remove `begin..rescue` wrapper around `require 'active_record'` on line 6.